mingw-w64-discount/PKGBUILD

```
@@ -3,11 +3,11 @@
 _realname=discount
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=2.2.7
+pkgver=2.2.7b
```

Why are you downgrading? 😛 
```
$ vercmp 2.2.7 2.2.8
-1

$ vercmp 2.2.7 2.2.7b
1
```

Alternative:
```suggestion
pkgver=2.2.7.b
```

```
$ vercmp 2.2.7 2.2.7.b
-1
```

---

Grr... Reverting the bump, the change is not significant anyway...